tests/libcxx/usual_matrix.lst

```diff
@@ -3,5 +3,6 @@
 
 PM_CL="/nologo /Od /W4 /w14061 /w14242 /w14265 /w14582 /w14583 /w14587 /w14588 /wd4643 /w14749 /w14841 /w14842 /w15038 /w15214 /w15215 /w15216 /w15217 /sdl /WX /Zc:strictStrings /D_ENABLE_STL_INTERNAL_CHECK /bigobj /FImsvc_stdlib_force_include.h /permissive-"
 RUNALL_CROSSLIST
-PM_CL="/EHsc /MTd /D_ITERATOR_DEBUG_LEVEL=2 /std:c++latest /analyze"
+# C5105: macro expansion producing 'defined' has undefined behavior is emitted by windows.h. TRANSITION, GH-854

```

Comment:
The lack of quotation marks here makes the boundaries unclear in these comments. I suggest one of:

1. `# C5105: "macro expansion producing 'defined' has undefined behavior" is emitted by Windows.h. TRANSITION, GH-854`
1. `# TRANSITION, GH-854; Windows.h emits C5105: "macro expansion producing 'defined' has undefined behavior"`
1. `# TRANSITION, GH-854` (We don't need explanation here since it's in the issue.)

My mild preference is for the third.

Reply:
I've created a merge conflict with #862, so I'll need to resolve that when merging - I will also push a change to update the TRANSITION comments as you suggest, as @barcharcraz is busy with `<bit>`.